src/styles/auto-prefix.js

```diff
@@ -20,18 +20,29 @@ export default {
     let prefixer = prefixers[userAgent];
     // None found, create a new instance
     if (!prefixer) {
-      prefixer = new InlineStylePrefixer(userAgent);
+      prefixer = new InlineStylePrefixer({userAgent: userAgent});
```

Comment:
@oliviertassinari Do we have any code formatting rules (or future eslinting plans) for this scenario?

If a key and value variable name both have the same name, should we just write it like this:
```js
  prefixer = new InlineStylePrefixer({userAgent});
```